schemas/common_definitions.json

```diff
@@ -18,20 +18,21 @@
       "type": "object",
       "required": [
         "title",
-        "content"
+        "contents"
       ],
       "properties": {
         "title": {
           "type": "string"
         },
-        "content": {
-          "$ref": "common_definitions.json#/content"
+        "contents": {
+          "$ref": "common_definitions.json#/contents"
         }
       },
       "additionalProperties": false
     }
   },
   "guidance": {
+    "description": "Answer guidance",
```

Comment:
We should rename this definition from `guidance` to `answer_guidance` to differentiate it from question guidance.